tests/openassetio/test/managerValidator/conftest.py

```diff
@@ -19,18 +19,33 @@
 
 # pylint: disable=invalid-name,redefined-outer-name
 # pylint: disable=missing-function-docstring
+
+import os
 from unittest import mock
 
 import pytest
 
 from openassetio import hostAPI, logging
-from openassetio.test.managerValidator import validatorHarness
 
 
 @pytest.fixture
-def mock_host_interface():
-    return mock.create_autospec(
-        validatorHarness.ValidatorHarnessHostInterface, instance=True, spec_set=True)
+def test_resources_dir():
```

Comment:
[consider] Here and below, ideally don't prefix fixtures with `test_`, it's a bit confusing.

Reply:
Oh man, yes it so is! 🤣  How about [this](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/184/commits/69c5b5abacfb9aa5e730ce687edcd002b0753317)?